frontend/projects/ui/src/app/pages/apps-routes/app-show/components/app-show-health-checks/app-show-health-checks.component.html

```diff
@@ -3,7 +3,7 @@
 >
   <ng-container *ngIf="checks.length">
     <ion-item-divider>Health Checks</ion-item-divider>
-    <ng-container *ngIf="connectionFailure; else connected">
+    <ng-container *ngIf="!!(disconnected$ | async); else connected">
```

Comment:
Since it's `ngIf` and it just tests for faultiness, it's unnecessary to cast it to `boolean`, just like inside `if (statement)` in code.